blogService/src/main/java/com/blog/blogService/controler/UserController.java

```diff
@@ -1,16 +1,16 @@
-package blogService.controler;
+package com.blog.blogService.controler;
 
-import blogService.dto.BlogDto;
-import blogService.dto.UserDto;
-import blogService.service.UserService;
+import com.blog.blogService.dto.UserDto;
+import com.blog.blogService.service.impl.UserServiceImpl;
 import lombok.AllArgsConstructor;
 import org.springframework.validation.annotation.Validated;
 import org.springframework.web.bind.annotation.*;
 
 @AllArgsConstructor
 @RestController
 public class UserController {
-    private final UserService userService;
+
+    private final UserServiceImpl userService;
```

Comment:
po to mamy interfejsy aby je wstrzykiwać, a NIE konkretne implementację. proszę zmienić!

Reply:
zrobione